src/build.py

```diff
@@ -755,6 +755,8 @@ def LLVM():
              '-DCOMPILER_RT_INCLUDE_TESTS=OFF',
              '-DLLVM_BUILD_LLVM_DYLIB=%s' % build_dylib,
              '-DLLVM_LINK_LLVM_DYLIB=%s' % build_dylib,
+             # Our mac bot's toolchain's ld64 is too old for trunk libLTO.
+             '-DLLVM_TOOL_LTO_BUILD=OFF',
```

Comment:
Assuming this is fine to disable on all platforms because we don't use LLVM's LTO anyway?